docs/developers/node-tutorial.md

```diff
@@ -0,0 +1,218 @@
+# Getting and Sending Transactions with Celestia Node
+
+In this tutorial we will cover how to use the Celestia Node API to submit and
+retrieve transactions from the Data Availability Layer by their namespace
+merkle tree.
+
+## Hardware Requirements
+
+You can find hardware requirements [here](https://docs.celestia.org/nodes/bridge-validator-node#install-golang).
+
+## Setting Up Dependencies
+
+You can follow the tutorial for setting up the dependencies [here](https://docs.celestia.org/nodes/bridge-validator-node#setting-up-your-bridge-node).
+
+## Installing Celestia App
+
+In order to begin, we would need to compile the Celestia App binary.
+
+```sh
+cd $HOME
+git clone https://github.com/celestiaorg/celestia-app.git
+cd celestia-app/
+git checkout tags/v0.4.0 -b v0.4.0
+make install
+```
+
+For more information, you can follow the steps on compiling the binary [here](https://docs.celestia.org/nodes/bridge-validator-node#install-celestia-app).
+
+Once `celestia-appd` is compiled, you can run a useful script for instantiating
+a single-node network.
+
+In the `contrib` directory, open up the file `single-node.sh` and replace its
+content with the following script:
+
+```sh
+#!/bin/sh
+
+set -o errexit -o nounset
+
+CHAINID="devnet-2"
+app="./celestia-appd"
```

Comment:
are we using the directory as a binary here?

Reply:
Likely a relic from my own script.

Reply:
They should have $app point to the binary or take it as an arg in the script.